recipes/sesh/recipe.yaml

```
@@ -0,0 +1,48 @@
+context:
+  name: sesh
+  version: "2.15.0"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/joshmedeski/${{ name }}/archive/v${{ version }}.tar.gz
+  sha256: bd74d4ef697fd186ee74b76f2100b16cb5ed2eb43d5e478c4c5481b659196d4d
+
+build:
+  number: 0
+  skip:
+    - win
+  script:
+    - go build -trimpath -ldflags "-s -w" -o $PREFIX/bin/
```

```suggestion
    - go build -ldflags "-s -w" -o $PREFIX/bin/
```
`-trimpath` should now automatically be set by `GOFLAGS` environment variable, so it's redundant to put it as an argument here.